<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sander-notebook" name="nl.xillio.xill.plugins.list.constructs.ContainsConstructTest" tests="3" failures="0" timestamp="21 Oct 2015 07:55:15 GMT" time="0.181" errors="0">
  <testcase name="testProcessInvalidTypeCheck" time="0.001" classname="nl.xillio.xill.plugins.list.constructs.ContainsConstructTest"/>
  <testcase name="testProcessCheckExistingValueInList" time="0.169" classname="nl.xillio.xill.plugins.list.constructs.ContainsConstructTest"/>
  <testcase name="testProcessCheckExistingValueInObject" time="0.011" classname="nl.xillio.xill.plugins.list.constructs.ContainsConstructTest"/>
</testsuite> <!-- nl.xillio.xill.plugins.list.constructs.ContainsConstructTest -->
